lib/modules/keyboardNav.js

```diff
@@ -1219,7 +1219,13 @@ function reply(selected = getSelected()) {
 	));
 }
 
-function navigateTo(newWindow, href) {
+function navigateTo(newWindow, href, leaveNP = false) {
```

Comment:
Since I suspect this function is headed towards adding more options, it would be nice to reorder the function signature into an options dictionary (keyword args) pattern. 

So, if you write it like this (or with flow typing and drop the false default)
```javascript
function navigateTo(href, { newWindow: false, leaveNP: false })
```

Then you can invoke it like this:


```javascript
 navigateTo('/example', { leaveNP: true })
```

Reply:
Commit #59e628 changes the signature as recommended